src/Storage/FlysystemStorage.php

```diff
@@ -67,7 +67,12 @@ protected function doResolvePath(PropertyMapping $mapping, ?string $dir, string
             return $path;
         }
 
-        return (string) $fs->getAdapter()->applyPathPrefix($path);
+        $adapter = $fs->getAdapter();
+        if (is_callable([$adapter, 'applyPathPrefix'])) {
```

Comment:
please add native function invocation.
Ideally, you should run php-cs-fixer on your core before committing it.